SECURITY.md

```diff
@@ -36,6 +36,6 @@ We prefer all communications to be in English.
 
 ## Policy
 
-Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
+Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/msrc/cvd).
```

Comment:
The SECURITY.md content is the verbatim text all new repos are getting seeded with (there's a comment tag at the top and bottom of the file indicating the text version).

For now, please revert the changes to this file.

Reply:
Will do

Reply:
If we revert this the checks will fail. I think we should go ahead and update this as the link is definitely incorrect.